x-pack/plugins/triggers_actions_ui/public/application/sections/rule_form/rule_form.tsx

```diff
@@ -328,6 +333,18 @@ export const RuleForm = ({
     []
   );
 
+  const initRuleNameInputRef = (target: HTMLInputElement | null) => {
```

Comment:
Thanks for trying to address this! I think this is part of a larger problem where the entire rule form re-renders with any change to the rule config or params and unfortunately, adding this debounce seems to be making this issue more obvious with the action connector SVGs.

https://user-images.githubusercontent.com/13104637/168642882-caf35018-2e01-43dd-a210-7e7eb333606b.mp4

This flickering also happens on `main` but it is less prominent, probably because it is re-rendering *more* frequently. WDYT of backing out these changes for now and the framework will open a separate issue to address?

Reply:
Seems like there's already an issue for that? https://github.com/elastic/kibana/issues/104881

Reply:
Yes! You are correct thank you for linking it.

Reply:
Fix removed